### PR TITLE
Improve error details

### PR DESCRIPTION
**Pull request by thallesfreitaszup**

### CODE REVIEW

**Review by cpgo — APPROVED** [review]

File: circle-matcher/src/main/java/io/charlescd/circlematcher/handler/ExceptionUtils.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
+package io.charlescd.circlematcher.handler;
```
Comment (lucasbfernandes) [inline circle-matcher/src/main/java/io/charlescd/circlematcher/handler/ExceptionUtils.java:1]: Insert license header

File: circle-matcher/src/test/groovy/io/charlescd/circlematcher/handler/DefaultErrorResponseTest.groovy [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package io.charlescd.circlematcher.handler
```
Comment (lucasbfernandes) [inline circle-matcher/src/test/groovy/io/charlescd/circlematcher/handler/DefaultErrorResponseTest.groovy:1]: Insert license header

File: circle-matcher/src/test/groovy/io/charlescd/circlematcher/handler/ErrorHandlerTest.groovy [hunk 3]
```diff
@@ -1,16 +1,172 @@
 package io.charlescd.circlematcher.handler
```
Comment (lucasbfernandes) [inline circle-matcher/src/test/groovy/io/charlescd/circlematcher/handler/ErrorHandlerTest.groovy:1]: Insert license header

File: moove/api/src/main/kotlin/io/charlescd/moove/api/MooveExceptionHandler.kt [hunk 4]
```diff
@@ -16,13 +16,12 @@
```
Comment (lucasbfernandes) [inline moove/api/src/main/kotlin/io/charlescd/moove/api/MooveExceptionHandler.kt:16]: License header

**Review by lucasbfernandes — CHANGES_REQUESTED** [review]

File: circle-matcher/src/main/java/io/charlescd/circlematcher/handler/ExceptionUtils.java (hunk 1 above)
Comment (thallesfreitaszup) [reply circle-matcher/src/main/java/io/charlescd/circlematcher/handler/ExceptionUtils.java:17]: done

File: circle-matcher/src/test/groovy/io/charlescd/circlematcher/handler/DefaultErrorResponseTest.groovy (hunk 2 above)
Comment (thallesfreitaszup) [reply circle-matcher/src/test/groovy/io/charlescd/circlematcher/handler/DefaultErrorResponseTest.groovy:17]: done

File: circle-matcher/src/test/groovy/io/charlescd/circlematcher/handler/ErrorHandlerTest.groovy (hunk 3 above)
Comment (thallesfreitaszup) [reply circle-matcher/src/test/groovy/io/charlescd/circlematcher/handler/ErrorHandlerTest.groovy:17]: done

File: moove/api/src/main/kotlin/io/charlescd/moove/api/MooveExceptionHandler.kt (hunk 4 above)
Comment (thallesfreitaszup) [reply moove/api/src/main/kotlin/io/charlescd/moove/api/MooveExceptionHandler.kt:16]: this file has license

File: circle-matcher/src/main/java/io/charlescd/circlematcher/handler/ExceptionUtils.java [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public class ExceptionUtils {
+
+    public static DefaultErrorResponse createNotFoundErrorResponse(String message, String sourceString) {
```
Comment (eliasalveszup) [inline circle-matcher/src/main/java/io/charlescd/circlematcher/handler/ExceptionUtils.java:27]: This method could be replaced by something like (the same for `createBadRequestError` and `createInternalServerError`:

`
public static DefaultErrorResponse createNotFoundErrorResponse(String message, String sourceString) {
     return createBusinessExceptionError(message, "Not found", sourceString, 404); 
}
`

File: circle-matcher/src/main/java/io/charlescd/circlematcher/handler/ExceptionUtils.java [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+                source,
+                getMetaInfo()
+        );
+    }
+
+    public static DefaultErrorResponse createBusinessExceptionError(
+            String message,
+            String title,
+            String sourceString,
+            Exception exception) {
```
Comment (eliasalveszup) [inline circle-matcher/src/main/java/io/charlescd/circlematcher/handler/ExceptionUtils.java:57]: `exception` parameter is not used

File: circle-matcher/src/main/java/io/charlescd/circlematcher/handler/ExceptionUtils.java (hunk 5 above)
Comment (thallesfreitaszup) [reply circle-matcher/src/main/java/io/charlescd/circlematcher/handler/ExceptionUtils.java:27]: done

File: circle-matcher/src/main/java/io/charlescd/circlematcher/handler/ExceptionUtils.java (hunk 6 above)
Comment (thallesfreitaszup) [reply circle-matcher/src/main/java/io/charlescd/circlematcher/handler/ExceptionUtils.java]: removed.Thanks!

**Review by eliasalveszup — APPROVED** [review]

**Review by cpgo — DISMISSED** [review]

**Review by eliasalveszup — APPROVED** [review]

**Review by lucasbfernandes — APPROVED** [review]

**Pull request merged**